cli/src/main/kotlin/Main.kt

```diff
@@ -95,10 +109,14 @@ object Main : CommandWithHelp() {
 
         showVersionHeader(jc.parsedCommand)
 
-        return if (version) 0 else run(jc)
+        val config = configFile?.expandTilde()?.let {
```

Comment:
> If `configFile` is not specified, shouldn't we default to `getUserOrtDirectory().resolve("ort.yml")`?

I will add the default location in a follow-up PR.

> What happens here if the specified file does not exist?

It was silently ignored because I forgot to move the require block from the old code. Fixed.